### ISSUE TITLE
First letter in field name changed to lower case

### ISSUE THREAD

**Issue by GianhTran** [apollographql/apollo-kotlin#1560]
#### Issue description
If a field name starts with a capital letter then Apollo changes it to a lower case letter. This causes the query or mutation to fail.

#### Additional details / screenshot

```
Version: apollo_version = '1.1.1'
Language: kotlin_version = "1.3.31"
```

<img width="301" alt="Screen Shot 2019-08-30 at 7 59 46 PM" src="https://user-images.githubusercontent.com/20894740/64022607-cd1c6580-cb60-11e9-8a43-92490900e58d.png">

<img width="696" alt="Screen Shot 2019-08-30 at 7 56 02 PM" src="https://user-images.githubusercontent.com/20894740/64022406-541d0e00-cb60-11e9-8be6-f09a59093339.png">

**Comment by sav007** [comment]
Could you please provide the full version of generated  `UpdateSavedPlaceMutation` model?

**Comment by GianhTran** [comment]
@sav007 for security reasons, I can not share the full part, so sorry about that, We decided to change the param name, so it works as well now, but I hope this bug will be fixed in next release

**Comment by ralphie9224** [comment]
I'll look into this one of no one else has.

**Comment by martinbonnin** [comment]
This one should be fixed with [3.0.0-rc01](https://www.charlesproxy.com/) @GianhTran can you try and let us know?

**Comment by martinbonnin** [comment]
I'm going to go ahead and assume this is now working.
